### PR TITLE
chore: add verification account

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationPresenter.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        
+        let sections: [Models.Section] = [
+            .verificationLevel
+        ]
+        
+        let sectionRows: [Models.Section: [Any]] = [
+            // TODO: localize!
+            .verificationLevel: [
+                VerificationViewModel(title: .text("Level 1"),
+                                      status: .verified,
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationPresenter.swift]: please pass the required statuses from interactor...
so when we connect to BE we only need to update the interactor

File: brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationVIP.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+protocol AccountVerificationViewActions: BaseViewActions, FetchViewActions {
+}
+
+protocol AccountVerificationActionResponses: BaseActionResponses, FetchActionResponses {
+}
+
+protocol AccountVerificationResponseDisplays: AnyObject, BaseResponseDisplays, FetchResponseDisplays {
+}
+
+protocol AccountVerificationDataStore: BaseDataStore, FetchDataStore {
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationVIP.swift:20]: the store will have to store the verification statuses

File: brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationInteractor.swift [hunk 3]
```diff
@@ -13,7 +13,18 @@ class AccountVerificationInteractor: NSObject, Interactor, AccountVerificationVi
 
     // MARK: - AccountVerificationViewActions
     func getData(viewAction: FetchModels.Get.ViewAction) {
-        presenter?.presentData(actionResponse: .init(item: Models.Item(title: "Under construction", image: "earth")))
+        let verificationItems = [
+            VerificationViewModel(title: .text("Level 1"),
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationInteractor.swift]: i didnt mean the whole object.. just the parts, that we will get from the BE 
(that will prolly be only status)

File: brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationVIP.swift [hunk 4]
```diff
@@ -18,6 +18,9 @@ protocol AccountVerificationResponseDisplays: AnyObject, BaseResponseDisplays, F
 }
 
 protocol AccountVerificationDataStore: BaseDataStore, FetchDataStore {
+    var verified: VerificationStatus? { get set }
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationVIP.swift]: not sure but i think the BE will only return 1 status.. not 3

Comment (dangelovska) [reply brd-ios/breadwallet/src/Redesign/Scenes/AccountVerification/AccountVerificationVIP.swift]: yes, I already updated it :)

**Review by rcresnik — APPROVED** [review]

**Pull request merged**